phpstan-baseline.neon

```
@@ -0,0 +1,6 @@
+parameters:
+    ignoreErrors:
+        -
```

We try to add a comment to explain why we ignore an error.

Also, I'm curious why we need to ignore this, configuration was always fine for others projetcs.

---

I don't know why this configuration is working without errors in other projects.

---

This issue was resolved after installing "phpstan/phpstan-symfony".
Thank you.